doc/API.md

```diff
@@ -57,7 +159,7 @@ struct {
 'target' is the hash that needs to be met by a block for the block to be valid.
 The target is inversely proportional to the difficulty.
 
-#### /consensus/block [GET]
+#### /consensus/blocks/:height [GET]
```

Comment:
yeah originally I was using the same syntax as `httprouter`, but I felt it would be too confusing to have both `:` and `*`, so I switched to `<>` for both.